generate.sh

```diff
@@ -88,6 +88,14 @@ ENVVARS="$(python ${BASE}/modules/scripts/config.py "${USERNAME}" "${REPONAME}")
 echo -e "\n${ENVVARS}\n" >> ${LOGFILE}
 eval $ENVVARS
 
+if [ -n "$TARGET_BRANCH" ]; then
+  git checkout "$TARGET_BRANCH" >> ${LOGFILE}
```

Comment:
just suggestion. have master as a default branch. users can override it. instead of checkout clone directly the branch only